.github/workflows/codeql-analysis.yml

```diff
@@ -56,10 +59,18 @@ jobs:
         # Prefix the list here with "+" to use these queries and those in the config file.
         # queries: ./path/to/local/query, your-org/your-repo/queries@main
 
+    - name: Setup .NET Core SDK
+      uses: actions/setup-dotnet@v3.0.3
+      with:
+        dotnet-version: ${{ env.dotnet_sdk_version }}
+
+    - name: Build
+      run: dotnet build
```

Comment:
Just to check, shouldn't we pass `-c Release` flag here?

Reply:
I have no idea TBH.
@roji?

Reply:
Thinking about it, codeql analysis should probably check the release build, so this makes sense to me.
I'll add it.

Reply:
Yeah, I don't think it's very important - but having codeql analyze the thing we're actually delivering to users is probably better.